functions-nextjs/package.json

```diff
@@ -23,9 +23,10 @@
     "@babel/cli": "^7.4.4",
     "@babel/core": "^7.4.4",
     "@babel/preset-env": "^7.4.4",
+    "@google-cloud/functions-emulator": "^1.0.0-beta.6",
```

Comment:
I'd rather include this as a suggestion in the Notes section of the Readme. Can we revert this one too